src/test/librados/io.cc

```diff
@@ -918,13 +918,13 @@ TEST_F(LibRadosIoEC, AppendRoundTrip) {
   char *buf3 = (char *)new char[alignment *2];
   int uasize = alignment/2;
   char *unalignedbuf = (char *)new char[uasize];
-  auto lam = [&] {
-            delete[] buf;
-            delete[] buf2;
-            delete[] buf3;
-            delete[] unalignedbuf;
-          };
-  scope_guard<decltype(lam)> sg(std::forward<decltype(lam)>(lam));
+  auto cleanup = [&] {
+    delete[] buf;
+    delete[] buf2;
+    delete[] buf3;
+    delete[] unalignedbuf;
+  };
+  scope_guard<decltype(cleanup)> sg(std::forward<decltype(cleanup)>(cleanup));
```

Comment:
nit, `sg(std::move(cleanup))` would suffice. we know the type of cleanup here. and we do want to rvalue reference here.